tests/contracts/integratees/MockChaiIntegratee.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.6.8;
+
+import "../PreminedToken.sol";
+import "./utils/MockIntegrateeBase.sol";
+
+contract MockChaiIntegratee is MockIntegrateeBase, PreminedToken("Chai", "CHAI", 18) {
```

Comment:
Interesting idea to just allow the defi protocol integratee to be an ERC20 token. This will work here for Chai since it's the only asset, but it wouldn't work for Compound or most other integrations b/c multiple tokens.

For the other protocols, we'll probably we'll need to keep a list of derivatives in the deployment script and seed the integratees with those just like primitives.

If that's the case, then it might be good to start that practice now so it's easy for the next devs working on derivative tests to use the same implementation.

What do you think?